### PR TITLE
Landing stripe

### PR DESCRIPTION
**Pull request by usirin**
/cc @devrim @cihangir

### CODE REVIEW

File: servers/lib/server/handlers/createteam.coffee [hunk 1]
```diff
@@ -234,6 +234,7 @@ afterGroupCreateKallback = (res, params) ->
       (fin) ->
         params = { sessionToken: client.sessionToken }
         params.coupon = coupon  if coupon
+        params.card = { token: stripeToken, default: true }  if stripeToken
```
Comment (usirin) [inline servers/lib/server/handlers/createteam.coffee]: /cc @cihangir

**Comment by sinan** [comment]
@usirin pull before push :)

**Comment by sinan** [comment]
img

**Comment by gokmen** [comment]
this should be optional for `default` env.

**Pull request closed without merging**